scripts/compare_strategies.py

```
@@ -0,0 +1,334 @@
+
+    # Build report
+    report = build_comparison_report(data_a, data_b, baselines_data)
+
+    # Generate markdown
+    markdown = generate_markdown(report)
+
+    # Output
+    if args.output_dir:
+        strategy_a = report["strategy_a"]
+        strategy_b = report["strategy_b"]
+        date_str = datetime.now().strftime("%Y-%m-%dT%H%M%S")
```

Ruff (0.15.10)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)